source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp

```diff
@@ -3715,10 +3715,18 @@ VariableSP SymbolFileDWARF::ParseVariableDIE(const SymbolContext &sc,
               type_sp->GetType()->GetByteSize().getValueOr(0),
               die.GetCU()->GetAddressByteSize());
 
+        // Swift let-bindings are marked by a DW_TAG_const_type.
+        bool is_constant = false;
+        if (IsSwiftLanguage(sc.comp_unit->GetLanguage())) {
+          DWARFDIE type_die = die.GetReferencedDIE(llvm::dwarf::DW_AT_type);
```

Comment:
I don't really like this. It creeps into a generic file. Can you think of a way to make this swift-specific?

Reply:
We don't have a good plugin mechanism for that at the moment. There are already 5 swift-specific extensions in this file in various places (e.g., https://github.com/apple/swift-lldb/blob/5db61e041f7b3d5e84bce27966e32b351c648946/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp#L3668)

Reply:
We could conceivably add a lenguange-specific IsConstant method here, but which object should it be on?

Reply:
Arguably doing this for *any* declared-as-const storage is not guaranteed to have the desired effect in most languages, including C/C++.